blocks/components/editable/index.js

```diff
@@ -103,10 +110,25 @@ export default class Editable extends wp.element.Component {
 
 	updateContent() {
 		const bookmark = this.editor.selection.getBookmark( 2, true );
-		this.editor.setContent( this.props.value );
+		this.setContent( this.props.value );
 		this.editor.selection.moveToBookmark( bookmark );
 	}
 
+	setContent( content ) {
+		if ( ! content ) {
+			return '';
+		}
+
+		content = wp.element.renderToString( content );
```

Comment:
Should we be concerned about performance here? `renderToString` on each render and `parse` on each `onChange` ?